docs/roadmap.md

```diff
@@ -0,0 +1,169 @@
+# Roadmap
```

Comment:
Is this a roadmap for 0.2.0, 1.0.0, or some other version?

Reply:
A proper roadmap with funding and dedicated team could set a deadline like "2018 Q1-Q2". That's how the vscode roadmap is organized https://github.com/Microsoft/vscode/wiki/Roadmap. However, Metals has no funding and the prioritization of tasks depends on personal motivation of our contributors. I prefer not to promise in what order or when any of this functionality will be implemented.

Reply:
I also think it shouldn't refer to any particular version. I think this roadmap is more of a "vision" document than a plan with milestones.

Reply:
Version numbers don't have to be tied to concrete dates. 

I think being able to say "we want to achieve X, Y, Z for 0.2.0, and A, B, C for 0.3.0" will allow us to better structure our work. In my opinion, a plan is useful regardless of whether we have funding or not.